[SYNTH-12971] Rename global to defaultTestOverrides (#1285)
DataDog/datadog-ci: src/commands/synthetics/README.md

  "tunnel": true
}
```

> **Note**: A `global` filed was present in the global configuration file. This field is now deprecated and replaced by `defaultTestOverrides`.

Reviewer: Suggestion:
```suggestion
**Note**: The `global` field from the global configuration file is now deprecated in favor of `defaultTestOverrides`.
```

Reviewer: But let's wait what the documentation team thinks

Reviewer 2: The suggestion looks good to me, but I'd remove the word "now"

  "tunnel": true
}
```

> **Note**: A `global` filed was present in the global configuration file. This field is now deprecated and replaced by `defaultTestOverrides`.

Reviewer: same here
